src/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -3434,126 +3315,23 @@ public object GetContextInfo()
             return new object[] { cumulClip, cumulTransform };
         }
 
-
-        public Region Clip
-        {
-            get
-            {
-                var region = new Region();
-                int status = SafeNativeMethods.Gdip.GdipGetClip(new HandleRef(this, NativeGraphics), new HandleRef(region, region._nativeRegion));
-                SafeNativeMethods.Gdip.CheckStatus(status);
-
-                return region;
-            }
-            set => SetClip(value, CombineMode.Replace);
-        }
-
-        public RectangleF ClipBounds
-        {
-            get
-            {
-                var rect = new GPRECTF();
-                int status = SafeNativeMethods.Gdip.GdipGetClipBounds(new HandleRef(this, NativeGraphics), ref rect);
-                SafeNativeMethods.Gdip.CheckStatus(status);
-
-                return rect.ToRectangleF();
-            }
-        }
-
-        public bool IsClipEmpty
-        {
-            get
-            {
-                int isEmpty;
-                int status = SafeNativeMethods.Gdip.GdipIsClipEmpty(new HandleRef(this, NativeGraphics), out isEmpty);
-                SafeNativeMethods.Gdip.CheckStatus(status);
-
-                return isEmpty != 0;
-            }
-        }
-
-        public RectangleF VisibleClipBounds
+        public RectangleF VisibleClipBounds		
         {
             get
             {
                 if (PrintingHelper is PrintPreviewGraphics ppGraphics)
                 {
                     return ppGraphics.VisibleClipBounds;
                 }
-
-                var rect = new GPRECTF();
+		
```

Comment:
nit: Extra whitespace here and below.

Reply:
@qmfrederik It looks like there is still some extra whitespace in this method. Could you clean it up?

Reply:
@mellinoe Oops.. hope I got it right this time.